bot/skills/covid_mode.py

```diff
@@ -409,17 +420,44 @@ def catch_message(update: Update, context: CallbackContext):
     infect_user_masked_condition(
         user_to_infect,
         INFECTION_CHANCE_MASKED,
-        INFECTION_CHANCE_UNMASKED)
+        INFECTION_CHANCE_UNMASKED,
+        context)
 
     prev_message_user = user
 
     _db.add(user)
 
-# @TODO
 
+def hashImg(img: bytearray) -> str:
+    sha1(img[-100:]).hexdigest()
+
+
+def is_avatar_has_mask(img: bytearray, context: CallbackContext) -> bool:
+    if (img is None) or len(img) < 100:
+        return False
+
+    # lookup existing value in cache
+    hash = hashImg(img)
+    isGood = context.chat_data['avatar_mask_cache'].get(hash)
```

Comment:
@C-Pro this is not Go!

Reply:
What's wrong?

Reply:
Oh camelCase :))))
I officially don't give a *uck

Reply:
@getjump https://github.com/getjump/vldc-bot/pull/2